src/widgets/Header/Nav/ui.tsx

```diff
@@ -3,10 +3,15 @@ import { FC } from 'react';
 import { StNav, StNavLink } from './styled';
 import { type TNav } from './typing';
 
-export const Nav: FC<TNav> = ({ navItems }) => (
-  <StNav $gap={52}>
+export const Nav: FC<TNav> = ({ navItems, curPathName }) => (
+  <StNav $gap={32}>
     {navItems.map((item, index) => (
-      <StNavLink key={`menu-item-${index}`} href={item.link} $justifyContent='center'>
+      <StNavLink
+        key={`menu-item-${index}`}
+        href={item.link}
+        $justifyContent='center'
+        className={item.link === curPathName ? 'active' : ''}
```

Comment:
```suggestion
        className={item.link === curPathName && 'active'}
```
Как вариант, чтобы `''` не писать

Reply:
TS не одобряет такой вариант
![image](https://github.com/agricultural-machinery-rental/client/assets/114286265/05dd6341-8d0b-4ddc-ab9a-eab581d69967)